components/carousel/events.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Events
+page_title: Events | Carousel for Blazor
+description: Events in the Carousel for Blazor.
+slug: carousel-events
+tags: telerik,blazor,carousel,events
+published: true
+position: 20
+---
+
+# Carousel Events
+
+This article describes the available events of the Telerik Carousel for Blazor.
```

Comment:
Add a bullet list with the event name - even if it is only one, so people can see what's available at a glance.